build-rv32e.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+INSTALL_DIR=./llvm16.0/
+mkdir -p $INSTALL_DIR
+
+
+cd llvm-project
+patch -p1 < ../patches/llvm-D70401.patch
+patch -p1 < ../patches/compiler-rt.patch
+
+
+mkdir -p build
+cd build
+cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR -DLLVM_ENABLE_PROJECTS='clang;lld' -DLLVM_TARGETS_TO_BUILD="X86;AArch64;RISCV;WebAssembly" -DLLVM_ENABLE_RUNTIMES=compiler-rt ../llvm
```

Comment:
Why X86 and Aarch64? That just bloats the libraries and therefore the solang binaries

Reply:
Good catch I removed them